specs/requests/teams_spec.py

```diff
@@ -49,6 +51,24 @@
             with it('saves the team'):
                 expect(Team.nodes.get_or_none(id_=self.response.json["id"], name="My Team")).to_not(be(None))
 
+        with description('notify dialogflow adapter'):
+            with before.each:
+                current_app.config['DIALOGFLOW_NOTIFY'] = True
+                self.DIALOGFLOW_URL = os.environ.get('DIALOGFLOW_ADAPTER') + '/teams'
+                self.rsps = responses.RequestsMock()
+                self.rsps.__enter__()
```

Comment:
This is honestly pretty hacky. If you want to leave it this way at least leave a comment on how it works. Or change it back to: `with responses.RequestsMock() rsps:`